tests/nativeapp/test_run_processor.py

```diff
@@ -742,10 +746,7 @@ def test_create_dev_app_recreate_app_when_orphaned(
             (None, mock.call("use role app_role")),
             (None, mock.call("use warehouse app_warehouse")),
             (
-                ProgrammingError(
-                    msg=ERROR_MESSAGE_093079,
-                    errno=93079,
-                ),
+                ProgrammingError(errno=93079),
```

Comment:
Missing constant? Let's add it? Same for all other errnos below.

Reply:
oops missed this one